source/Octopus.Server.Client/Model/TagResource.cs

```diff
@@ -36,9 +36,9 @@ public class TagResource : INamedResource
         [JsonProperty(Order = -50, NullValueHandling = NullValueHandling.Ignore)]
         public string CanonicalTagName { get; set; }
 
-        [Writeable]
-        [Trim]
-        public int SortOrder { get; set; }
+        public string FullyQualifiedSlug { get; set; }
+
+        [Writeable] [Trim] public int SortOrder { get; set; }
```

Comment:
Is this unintentional formatting that has gone whoopsie?

Reply:
Yeah looks like it (Thanks Rider?)
All fixed now 👍

Reply:
Aye yeah who knows! Nothing like ole code cleanup merge conflicting all day long.